src/components/NavBarTopics.jsx

```diff
@@ -0,0 +1,22 @@
+import { Link } from "react-router-dom";
+
+const NavBarTopics = () => {
+  const myTopics = ["coding", "cooking", "football"];
+
+  return (
+    <div className="topicsTitle">
+      <strong> Topics: </strong>
+      {myTopics.map((topic) => {
+        return (
+          <Link to={`/${topic}`}>
+            <li className="navBarTopics" key={topic}>
+              {topic}
+            </li>
+          </Link>
+        );
+      })}
+    </div>
+  );
+};
+
+export default NavBarTopics;

```

Comment:
Great to see you've managed to get this working Julian, really nice work. and good responsiveness in resizing